nipype/interfaces/ants/utils.py

```diff
@@ -225,3 +225,46 @@ class AffineInitializer(ANTSCommand):
 
     def _list_outputs(self):
         return {'out_file': os.path.abspath(self.inputs.out_file)}
+
+
+class ComposeMultiTransformInputSpec(ANTSCommandInputSpec):
+    dimension = traits.Enum(3, 2, argstr='%d', usedefault=True, mandatory=True,
+                            position=0, desc='image dimension (2 or 3)')
+    output_transform = File(argstr='%s', mandatory=True, position=1,
```

Comment:
Please use `name_source`:

```Python
        output_transform = File(argstr='%s', mandatory=True, position=1,
                                   name_source=['transforms'], name_template='%s_composed',
                                   desc='the name of the resulting transform.')
```

And remove member `_list_outputs` that will not be necessary anymore.